cheeses/managers.py

```diff
@@ -1,24 +1,57 @@
 from django.db import models
+from django.db.models import Count, Sum
 
 class CheeseQuerySet(models.QuerySet):
-    def get_users_cheese(self, username):
-        return self.filter(owner__username=username)
+    def get_cheeses_for_users(self, users):
+        return self.filter(owner__in=users)
     
-    def get_top_cheese(self, size):
-        return self.order_by('-reviews')
-
-    def test(self):
-        return self.filter(reviews__cheese=2)
+    def get_top_cheese(self, size=5):
+        return self.all().annotate(reviews_count=Count('reviews')).order_by('-reviews_count')[:size]
 
 class CheeseManager(models.Manager):
     def get_queryset(self):
         return CheeseQuerySet(self.model, using=self._db)
 
-    def get_users_cheese(self, username):
-        return self.get_queryset().get_users_cheese(username)
+    def get_cheeses_for_users(self, users):
+        return self.get_queryset().get_cheeses_for_users(users)
 
-    def get_top_cheese(self, size):
+    def get_top_cheese(self, size=5):
         return self.get_queryset().get_top_cheese(size)
 
-    def test(self):
-        return self.get_queryset().test()
+
+
+
+class ReviewQuerySet(models.QuerySet):
+    def get_reviews_for_score(self, score):
+        return self.filter(score=score)
+
+    def get_reviews_by_content(self, string):
+        return self.filter(content__icontains=string)
+
+    def get_top_review(self, cheese):
+        return self.filter(cheese__name=cheese).annotate(ratings_sum=Sum('ratings__vote')).order_by('-ratings_sum').first()
```

Comment:
you don't need `cheese__name=cheese`, when `cheese=cheese` should do the job;
I'm assuming that this should have thrown an error or not worked correctly

Reply:
you could have used `get_top_reviews` for getting the top review, by calling the method with `size=1`